packages/selected-network-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/selected-network-controller",
-  "version": "4.0.0",
+  "version": "4.1.0",
```

Comment:
It looks like there are breaking changes that have already been captured in the changelogs. Can we bump this to 5.0.0?